### PR TITLE
Update

### PR DESCRIPTION
**Pull request by kevinbleckmann**
Branding Update
Standard file naming change
updated Provider Version

### CODE REVIEW

File: Examples/EKS_Managed_Node_Group/README.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
+<p align="center">
+  <img src="https://github.com/OTCShare2/terraform-intel-aws-eks/blob/main/images/logo-classicblue-800px.png?raw=true" alt="Intel Logo" width="250"/>
+</p>
+
+# Intel® Cloud Optimization Modules for Terraform
+
+© Copyright 2022, Intel Corporation
```
Comment (zunairaahmad1) [inline Examples/EKS_Managed_Node_Group/README.md:7]: Need to change copyright year

File: Examples/EKS_Managed_Node_Group/README.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
+<p align="center">
+  <img src="https://github.com/OTCShare2/terraform-intel-aws-eks/blob/main/images/logo-classicblue-800px.png?raw=true" alt="Intel Logo" width="250"/>
+</p>
+
+# Intel® Cloud Optimization Modules for Terraform
```
Comment (zunairaahmad1) [inline Examples/EKS_Managed_Node_Group/README.md:5]: Need to Update Branding

File: Examples/Self_Managed_Node_Group/README.md [hunk 3]
```diff
@@ -0,0 +1,186 @@
+<p align="center">
+  <img src="https://github.com/OTCShare2/terraform-intel-aws-eks/blob/main/images/logo-classicblue-800px.png?raw=true" alt="Intel Logo" width="250"/>
+</p>
+
+# Intel® Cloud Optimization Modules for Terraform
```
Comment (zunairaahmad1) [inline Examples/Self_Managed_Node_Group/README.md:5]: Need to update branding

File: Examples/Self_Managed_Node_Group/README.md [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+
+© Copyright 2022, Intel Corporation
```
Comment (zunairaahmad1) [inline Examples/Self_Managed_Node_Group/README.md:7]: Need to update copyright year

Comment (kevinbleckmann) [reply Examples/Self_Managed_Node_Group/README.md]: fixed

File: Examples/Self_Managed_Node_Group/README.md (hunk 3 above)
Comment (kevinbleckmann) [reply Examples/Self_Managed_Node_Group/README.md]: fixed

File: Examples/EKS_Managed_Node_Group/README.md (hunk 1 above)
Comment (kevinbleckmann) [reply Examples/EKS_Managed_Node_Group/README.md]: fixed

File: Examples/EKS_Managed_Node_Group/README.md (hunk 2 above)
Comment (kevinbleckmann) [reply Examples/EKS_Managed_Node_Group/README.md]: fixed

**Review by mandalrajiv — APPROVED** [review]

**Pull request merged**